### ISSUE TITLE
[W10] Port 53682 Excluded

### ISSUE THREAD

**Issue by dolpsdw** [rclone/rclone#4688]
#### What is the problem you are having with rclone?
On latest Windows 10, with latest rclone beta.
When i try to make <rclone config> or <rclone authorize "amazon cloud drive">
The program fail with bind error on port 53682 because unauthorized.

Apparently Microsoft in one updated reserved some ports and dont authorize open conections on them.
(Not sure if W10 -> 10.0.18363.1139 related or Other software reserving that ports)

For a full list check in a cmd or powershell:
netsh interface ipv4 show excludedportrange protocol=tcp

After patching the port to other number like 53703, now the auth service bind correctly.

#### What is your rclone version (output from `rclone version`)
rclone v1.54.0-beta.4831.4e2b5389d
- os/arch: windows/amd64
- go version: go1.15.2

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
Windows 10 64

####  Which cloud storage system are you using? (eg Google Drive)
irrelevant

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
<rclone authorize "amazon cloud drive">

**Comment by animosity22** [comment]
Sounds like you have a firewall/av/etc blocking it.

![image](https://user-images.githubusercontent.com/5088712/96251421-70efed00-0f7e-11eb-9f16-ad8d47ba0f40.png)

**Comment by dolpsdw** [comment]
Its because the port is in excludedPortRange
![image](https://user-images.githubusercontent.com/4348598/96251942-79155000-0fb1-11eb-8720-e018b5e2ed4d.png)

**Comment by animosity22** [comment]
Right which is not default so you have a firewall/AV/Security/etc blocking it...

Here is a vanilla Windows 10 machine.

![image](https://user-images.githubusercontent.com/5088712/96252103-6f72f480-0f7f-11eb-9604-937c19c1168f.png)

**Comment by dolpsdw** [comment]
i am using default Windows defender antivirus.
If that ports are there its because some legis software have register them ?

Dont know the culprit.

**Comment by animosity22** [comment]
That means something / you / group policy / friend / another piece of software has blocked those ports.

**Comment by animosity22** [comment]
Going to close out the question since it's a local computer question. You can always give a post on the forums and perhaps someone else might have a suggestion as well:

https://forum.rclone.org

**Comment by lithiumfrost** [comment]
For those that run into this, at least on my box, this was not related to any firewall. Rather, it was the `winnat` that comes with Windows 10 that set excluded ports visible with this command. It's not another piece of software responsible; it's the OS itself.

```
netsh interface ipv4 show excludedportrange protocol=tcp
```

In order to run the rclone authorize command, I had to `net stop winnat` to make the port rclone is using available. Once I was done with the authorization, I restarted the Windows service. Took me a while to figure this one out.
